### PR TITLE
add highlights for 2022-10-06

### PR DESCRIPTION
**Pull request by weatherhog**
This Weeks Highlights PR

### CODE REVIEW

**Comment by T-Kukawka** [comment]
all good from Phoenix

**Comment by alex-dabija** [comment]
No updates from Hydra.

File: src/content/changes/highlights/2022-10-06.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+date: 2022-10-06T10:00:00
+title: Highlights for the week ending October 96, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
```
Comment (gianfranco-l) [inline src/content/changes/highlights/2022-10-06.md:9]:
```suggestion
## General

The customer cluster admin group and `default:automation` service account have now access to Flux logs for debugging issues with the flux system.

```

**Comment by gianfranco-l** [comment]
added contribution from honeybadger

**Pull request merged**